src/exchange/account/home/account-home.component.js

```diff
@@ -148,6 +165,19 @@
           for (let i = formattedAccounts.length; i < this.accounts.length; i += 1) {
             this.accounts.splice(i, 1);
           }
+
+          if (this.preferencesAlreadyExists) {
+            return this.$q.when();
```

Comment:
you can just ```return;```

Reply:
No I can't but returning `this.$q.when()` in a `Promise` is indeed non-sensical 😄